doc/policies/enrollment.rst

```diff
@@ -256,3 +256,21 @@ regular expression. During registration of the U2F device the information
 is fetched from the attestation certificate.
 Only if the attribute in the attestation certificate matches accordingly the
 token can be registered.
+
+.. _2step_parameters:
+
+2step_clientsize, 2step_serversize, 2step_difficulty
```

Comment:
These are actually called `{type}_2step_clientsize` etc

Reply:
Right. I was not sure about it, if I should write *all* policy actions (hotp and totp) or a tag or whatever. Will do the tag.